comdb2/cdb2.py

```diff
@@ -128,48 +128,79 @@
     Make sure to carefully read :ref:`String and Blob Types` on that page.
 """
 
+from __future__ import annotations
+
+import datetime
+from typing import Any, Callable, Iterator, List, Mapping, Tuple, Union
```

Comment:
I think a number of these should potentially come from `collections.abc`, but I'm not sure of whether they would be available from there for 3.8 for example.

Reply:
Well, as long as we use `from __future__ import annotations` we can use them from there anyway, and type checkers should be fine with it, I think...

Reply:
Then let's do that.